### PR TITLE
feat: allow to import require statements and show all failing with multi-error

### LINKED ISSUE #2353: Require should be added to ImportSections to allow recursive requirement checks
**Is your feature request related to a problem?**  
Yes. I'm trying to create a library of reusable devspace components that can be used for different project across the company. My goal is for example to create a common setup for all golang projects and import all the required components to keep the devspace.yaml within the projects as small as possible. This is to avoid to have to apply changes to similar projects one by one. While I was setting up things, I noticed that `require` is not imported recursive. To me this is some kind of important, as I want to define the requirements in the different components, that are using different commands and plugins.

**Which solution do you suggest?**  
I suggest to add `require` to the imported fields list here: https://github.com/loft-sh/devspace/blob/main/pkg/devspace/config/loader/imports.go#L17
I already prepared a Pull-Request for that. Which I will reference to this issue soon.

**Which alternative solutions exist?**  

**Additional context**  

/kind feature

### PR DESCRIPTION
**Pull request by dtomasi**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind enhancement
/kind feature

**What does this pull request do? Which issues does it resolve?** (use `resolves #<issue_number>` if possible) 
resolves #2353

**Please provide a short message that should be published in the DevSpace release notes**
Fixed an issue where DevSpace does not resolve `require` recursive in `imports`. Additionally this introduces mutlierror for requirements to output all unmet requirements at once. This is useful if there are a bunch of imports with different requirements.

**What else do we need to know?**

### CODE REVIEW

**Review by lizardruss — CHANGES_REQUESTED** [review]
Thanks for the PR!

See my inline comments for requested changes

File: pkg/devspace/config/loader/loader.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/loft-sh/devspace/pkg/devspace/plugin"
 	"github.com/loft-sh/devspace/pkg/devspace/upgrade"
 
+	"github.com/hashicorp/go-multierror"
```
[the remark this answers is not in the document]
Comment (dtomasi) [reply pkg/devspace/config/loader/loader.go]: @lizardruss thanks. wasn't aware of the kube package.

**Review by FabianKramm — APPROVED** [review]
@dtomasi thanks for the PR! LGTM!

**Comment by FabianKramm** [comment]
@dtomasi unfortunately the tests are failing, please take a look there!

**Comment by dtomasi** [comment]
@FabianKramm well ... a stupid mistake in variable assignment caused this. Unfortunately, I cannot see if everything passes now now, as of

> First-time contributors need a maintainer to approve running workflows. [Learn more.](https://docs.github.com/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)

However ... Hope this fixes the tests.

**Review by lizardruss — APPROVED** [review]

**Pull request merged**